react-app/src/modules/account/account.js

```diff
@@ -0,0 +1,100 @@
+import scrypt from 'scrypt-async';
+import nacl from 'tweetnacl';
+import sha256 from 'fast-sha256';
+import blake from 'blakejs';
+
+nacl.util = require('tweetnacl-util');
+
+let _username, _pk, _sk, _tz, _offset;
+
+export function clearAccount() {
+  _username = _pk = _sk = _tz = _offset = undefined;
+}
+
+export function setKeyPair(pk, sk) {
+  _pk = nacl.util.decodeBase64(pk);
+  _sk = nacl.util.decodeBase64(sk);
+}
+
+function _generateNonce(publicKeyA, publicKeyB) {
+  let state = blake.blake2bInit(nacl.box.nonceLength, null);
+	blake.blake2bUpdate(state, publicKeyA);
+  blake.blake2bUpdate(state, publicKeyB);
+  return blake.blake2bFinal(state);
+}
+
+export function constructor(username, password) {
```

Comment:
This may be a dumb question, but couldn't you do something like what's defined here: https://javascript.info/class to define an actual class?

Reply:
Yes. The idea is to manage the account as a class therefore? I'll implement this changes.

Reply:
@ivanalejandro0 should decide this. I think the conversation ended in: implement Account as a class, export an instance of Account. But now I'm not sure.

Reply:
Ivan just confirmed me that class + export instance is the way to go.